jetty-home/src/main/resources/modules/annotations.mod

```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT THIS FILE - See: https://jetty.org/docs/
+
+[description]
+Enables Annotation parsing.
```

Comment:
Feels like this should be in `jetty-core/jetty-annotations/src/main/config/modules`, like we do with the other modules.

Reply:
moved